docker-compose.yaml

```diff
@@ -44,6 +50,36 @@ services:
             - ./backend/supervisor.d/:/etc/supervisor/conf.d/
             # uncomment after Step 2 of README
             #- ./local_configs/backend/supervisor.d/:/etc/supervisor/conf.d/
+    monitor:
+        image: inspiregroup/artemis-monitor:${SYSTEM_VERSION}
+        container_name: monitor
+        depends_on:
+            - rabbitmq
+        networks:
+            - artemis
+        expose:
+            - ${MON_SUPERVISOR_PORT}
+        environment:
+            RABBITMQ_USER: ${RABBITMQ_USER}
+            RABBITMQ_PASS: ${RABBITMQ_PASS}
+            RABBITMQ_HOST: ${RABBITMQ_HOST}
+            RABBITMQ_PORT: ${RABBITMQ_PORT}
+            RIS_ID: ${RIS_ID}
+            MON_SUPERVISOR_HOST: ${MON_SUPERVISOR_HOST}
+            MON_SUPERVISOR_PORT: ${MON_SUPERVISOR_PORT}
+            REDIS_HOST: ${REDIS_HOST}
+            REDIS_PORT: ${REDIS_PORT}
+        volumes:
+            # uncomment to run from source code (only if you build from source)
+            # - ./monitor/:/root/
+            # comment after Step 2 of README
+            - ./monitor/configs/:/etc/artemis/
+            # uncomment after Step 2 of README
+            # - ./local_configs/backend/:/etc/artemis/
+            # comment after Step 2 of README
+            - ./monitor/supervisor.d/:/etc/supervisor/conf.d/
+            # uncomment after Step 2 of README
+            #- ./local_configs/backend/supervisor.d/:/etc/supervisor/conf.d/
```

Comment:
this line needs to be removed (nobody should change the supervisor config for monitors, and it is fine that only the backend is in the default local_configs (which still needs to be tuned at will).